### DISCUSSION TITLE
D97A Orders

### DISCUSSION THREAD

**Discussion by a-link-coder** [php-edifact/edifact#127] [Q&A]
Hello, 

i have the problem, that the edifile we get, has multipile Line items, however, i found out that the maxreapet in the d97b file for orders says maxrepeat 1.

I copied and changed maxrepeat to 999, verfied that the new file is being loaded.
But I still only get one lineitem.

pls send help.

**Comment by netbrothers-tr** [comment]
I think, you will have to provide the file (with secrets redacted) and also some code in order for the community to help you. Otherwise, one can neither fully understand the problem nor reproduce it in any way.

**Comment by a-link-coder** [comment]
The Problem was, that the EDI FIle had more than 25 PIA, thus, the interpreter seems to just stop right there.

**Reply by sabas** [reply]
If you go over the max repeat you should get an error logged in the array
